spec/services/streams/twitch_access_token/get_spec.rb

```diff
@@ -33,7 +33,7 @@
 
     context "when there is an expired token in the cache" do
       it "requests a new token and caches it" do
-        RedisRailsCache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, ["FAKE_EXPIRED_TWITCH_TOKEN", 15.days.ago])
+        RedisRailsCache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, "FAKE_EXPIRED_TWITCH_TOKEN", :expires_in => 15.days.from_now)
```

Comment:
This is setting the expiration in the future but this test is testing what happens when a token is expired.
> "when there is an expired token in the cache"
In this case, an expired token will not exist, in other words, given no token in the cache(by removing this line) then we should return a brand new one.

Reply:
I really didn't get that, So this line is not needed any more 

```
RedisRailsCache.write(described_class::ACCESS_TOKEN_AND_EXPIRATION_CACHE_KEY, "FAKE_EXPIRED_TWITCH_TOKEN", :expires_in => 15.days.from_now)
```

Reply:
or will `15.days.ago work` ?

Reply:
We want to remove the write line. Creating a token in Redis with an expiration in the past doesn't really make sense bc it will just expire itself immediately and not exist. What we want to test here is what happens when a token has expired, ie does not exist in Redis. One of the easiest ways to not have it exist in Redis is to not write it in the first place.